packages/client/src/modules/translations/resources/en/common.json

```diff
@@ -30,11 +30,11 @@
   "cta.read-profile-oilgre": "View Oil'gre persona",
   "cta.read-synthesis": "Access synthesis",
   "cta.read-terms-of-use": "Lire les conditions d'utilisation",
```

Comment:
C'est normal qu'il y ait que la moitié qui est traduite?

Reply:
It is weird

Reply:
J'ai traduit que les clés qui sont effectivement utilisées côté joueur comme j'ai besoin de vérifier la pertinence de la traduction à chaque fois. Celle là n'est plus utilisée, je la supprime